python/sbp/client/framer.py

```diff
@@ -101,29 +104,33 @@ def _receive(self):
     # empty input
     preamble = self._read(1)
```

Comment:
Yes already changed here:
https://github.com/swift-nav/libsbp/pull/473/files#diff-a50e439941512d511239e80cb8d1127dR79

You can check from the profiler results line for `PySerialDriver.read`

After this the deciding factor is the serial port timeout which is a trade-off between CPU and console "real-timeness" as I explained here https://github.com/swift-nav/libsbp/pull/473#issuecomment-305118567